evap/student/views.py

```diff
@@ -41,7 +41,7 @@ def index(request):
         evaluation.participates_in = request.user in evaluation.participants.all()
         evaluation.voted_for = request.user in evaluation.voters.all()
     evaluations = get_evaluations_with_course_result_attributes(evaluations)
-    evaluations.sort(key=lambda evaluation: evaluation.full_name)  # evaluations must be sorted for regrouping them in the template
+    evaluations.sort(key=lambda evaluation: (evaluation.course.name, evaluation.name))  # evaluations must be sorted for regrouping them in the template
```

Comment:
why do we not want to use `full_name` here?

Reply:
it has the exact same result in the end. however, the regroup in the template requires the evaluations to be sorted by course, and i thought this would make it more explicit, instead of using the human-readable name for a technical requirement that just happens to work. I don't have strong feelings here though :)